go.mod

```diff
@@ -62,7 +62,7 @@ require (
 	go.opentelemetry.io/otel/metric v1.35.0 // indirect
 	go.opentelemetry.io/otel/sdk v1.19.0 // indirect
 	go.opentelemetry.io/otel/trace v1.35.0 // indirect
-	golang.org/x/net v0.28.0 // indirect
+	golang.org/x/net v0.38.0 // indirect
```

Comment:
question: is this change needed for your changes? I ask because any update to the core's go.mod file needs to trigger an update in all modules' go.mod files.

Reply:
Sorry, this was not needed. Don't know why this was included in my PR.